buildexo/build_exo.py

```diff
@@ -0,0 +1,69 @@
+import subprocess
+import sys
+import os 
+def run():
+
+    python_version = f"python{sys.version_info.major}.{sys.version_info.minor}"
+
+    command = [
+        f"{sys.executable}", "-m", "nuitka", "exo/main.py",
+        "--company-name=exolabs",
+        "--product-name=exo",
+        "--output-dir=dist",
+        "--follow-imports",
+        "--standalone",
+        "--output-filename=exo",
+        "--static-libpython=yes"
+    ]
+
+    if sys.platform == "darwin": 
+        command.extend([
+            "--macos-app-name=exo",
+            "--macos-app-mode=gui",
+            "--macos-app-version=0.0.1",
+            "--macos-create-app-bundle",
+            "--macos-app-icon=docs/exo-logo.icns",
+            "--include-module=exo.inference.mlx.models.llama",
+            "--include-module=exo.inference.mlx.models.deepseek_v2",
+            "--include-module=exo.inference.mlx.models.base",
+            "--include-module=exo.inference.mlx.models.llava",
+            "--include-module=exo.inference.mlx.models.qwen2",
+            "--include-distribution-meta=mlx",
+            "--include-module=mlx._reprlib_fix",
+            "--include-module=mlx._os_warning",
+            f"--include-data-files=./env/lib/{python_version}/site-packages/mlx/lib/mlx.metallib=mlx/lib/mlx.metallib",
```

Comment:
i think this is making assumptions about where your virtual env is located. I think maybe this should use site packages. For example:

```
site_packages = site.getsitepackages()[0]
...

and then replace `env` in the f-string with `{site_packages}`
```

(code above is paraphrased, not tested)

Reply:
correct. its using the site modules now. thanks for catching that!

Reply:
thanks for catching that @aiden-dnewman